api/openapi-spec/swagger.json

```diff
@@ -7358,14 +7358,7 @@
           "type": "string"
         },
         "type": {
-          "description": "Type of node condition.\n\nPossible enum values:\n - `\"DiskPressure\"` means the kubelet is under pressure due to insufficient available disk.\n - `\"MemoryPressure\"` means the kubelet is under pressure due to insufficient available memory.\n - `\"NetworkUnavailable\"` means that network for the node is not correctly configured.\n - `\"PIDPressure\"` means the kubelet is under pressure due to insufficient available PID.\n - `\"Ready\"` means kubelet is healthy and ready to accept pods.",
-          "enum": [
```

Comment:
what versions does this need to be backported to? this is a bug that can break generated clients.

Reply:
There is only 1.23 since the enum support was introduced.